Orange/util.py

```diff
@@ -0,0 +1,68 @@
+"""Various small utilities that might be useful everywhere"""
+
+from functools import wraps
+import sys
+import abc
+import numpy as np
+
+
+def scale(values, min=0, max=1):
+    """Return values scaled to [min, max]"""
+    ptp = np.nanmax(values) - np.nanmin(values)
+    if ptp == 0:
+        return np.clip(values, min, max)
```

Comment:
I didn't yet. My implementation also returns the same value for all constant array, but, obviously, by just clipping it to the selected range, keeps more information, however "worthless" it may be. Would centering be better? If I have an attribute of values 1–5, and a random subset is selected consisting only of 5's, then a range of 3's is returned (provided the scaling is done to [1, 5], for whatever reason), whereas 5's are kept in my case.